### PR TITLE
DM-43278: Removed support for the file-based empty chunk collections

### PR DESCRIPTION
**Pull request by iagaponenko**

### CODE REVIEW

File: src/css/EmptyChunks.cc [hunk 1]
```diff
@@ -37,43 +35,45 @@
...
-    lock_guard<mutex> lock(_setsMutex);
-    IntSetMap::const_iterator i = _sets.find(db);
-    if (i != _sets.end()) {
-        IntSetConstPtr readOnly = i->second;
-        return readOnly;
+EmptyChunks::EmptyChunks(map<string, set<int>> const& database2chunks) {
+    for (auto const& [db, chunks] : database2chunks) {
+        auto newSet = make_shared<set<int>>();
+        *newSet = chunks;
+        _sets.insert(make_pair(db, newSet));
```
Comment (jgates108) [inline src/css/EmptyChunks.cc:53]: I think this can be 
```_sets.insert({db, newSet});```

File: src/css/EmptyChunks.cc [hunk 2]
```diff
@@ -37,43 +35,45 @@
...
+
+shared_ptr<set<int> const> EmptyChunks::getEmpty(string const& db) {
+    lock_guard<mutex> lock(_mtx);
+    if (auto itr = _sets.find(db); itr != _sets.end()) {
+        return itr->second;
+    }
+    auto newSet = make_shared<set<int>>();
+    if (_databaseInterface != nullptr) {
+        *newSet = _populate(db);
+        _sets.insert(make_pair(db, newSet));
```
Comment (jgates108) [inline src/css/EmptyChunks.cc:65]: Same as above.

**Review by jgates108 — APPROVED** [review]
Looks good, just minor comments.

File: src/css/EmptyChunks.cc (hunk 1 above)
Comment (iagaponenko) [reply src/css/EmptyChunks.cc]: Good suggestion!

**Pull request merged**